tutorial-1-translating-with-jobs/nodejs/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "scripts": {
+    "build": "tsc"
+  },
+  "dependencies": {
+    "smartling-api-sdk-nodejs": "Smartling/api-sdk-nodejs#CON-992-updated-jobs-api-client",
```

Comment:
Needs to be changed once this PR is landed https://github.com/Smartling/api-sdk-nodejs/pull/58